definitions/procedures/backup/offline/foreman_db.rb

```diff
@@ -31,22 +31,47 @@ def run
       private
 
       def local_backup
-        with_spinner("Collecting data from #{pg_data_dir}") do
-          feature(:foreman_database).backup_local(
-            pg_backup_file,
-            :listed_incremental => File.join(@backup_dir, '.postgres.snar'),
-            :volume_size => @tar_volume_size,
-            :data_dir => pg_data_dir
-          )
+        with_spinner("Collecting data from #{pg_data_dirs.join(',')}") do
+          pg_data_dirs.each_with_index do |pg_dir, index|
+            if index == 0
+              do_backup(pg_dir, 'create')
+            else
+              do_backup(pg_dir, 'append')
+            end
```

Comment:
Nit: you could even reduce it down to:
```suggestion
            do_backup(pg_dir, index == 0 ? 'create' : 'append')
```